.github/ISSUE_TEMPLATE/NEW_TOPIC.md

```diff
@@ -0,0 +1,33 @@
+---
+name: New topic
+about: Create a request to add missing information to devdocs
+---
+
+## Description
+
+<!-- (REQUIRED) What is the topic missing from devdocs -->
+
+## Specifics to include
+
+<!-- (REQUIRED) List specific information or details to include in this topic -->
+
+[ ] Explanation for ...
+[ ] How to ...
```

Comment:
Change to Tutorial, or How to/Tutorial ...
We removed the How To formats and replaced them with Tutorials.

Reply:
section revised to make it clearer on how it should be filled out